ONSdigital/sdc-int-rh-ui
# 11 rhui remove address confirmation page and dependency on address

# Motivation and Context


# What has changed
Lots, removed loads and changed calling from  get UAC and confirm,  to pass in uac hash and get back tokens or error.  

# How to test?
make test. Then build it with docker-build,  and make up in docker dev (with new rh-service in place from ticket).  Populate some open UACs, receipted UACs, deactivated UACs with the ATs (I'd advise) - get the UACs and see if it works as expected.  

Try no UAC, should say: enter one
Try a bad UAC: it should say enter a valid one
Try an Inactive real UAC,  it should redirect to page saying: inactive UAC etc
Try a receipted UAC: should redirect to a page saying it's already been used.
Try a good UAC should redirect to localhost:5000?token=blah

# Links




# Screenshots (if appropriate):
